### DISCUSSION TITLE
move from rc to run-con

### DISCUSSION THREAD

**Discussion by goatandsheep** [sapegin/mrm#317] [Ideas]
The most popular fork of rc is currently [run-con](https://www.npmjs.com/package/run-con) which is being maintained whereas rc has been deprecated. Should we move to run-con?

**Comment by kevinkhill** [comment]
I would argue that [cosmiconfig is a much more robust solution than run-con](https://github.com/davidtheclark/cosmiconfig)

https://npmcompare.com/compare/cosmiconfig,rc,run-con

**Reply by cwtuan** [reply]
The second link is broken. Use [cosmiconfig vs rc vs run-con](https://npm-compare.com/cosmiconfig,rc,run-con) instead.

**Reply by goatandsheep** [reply]
I am the maintainer of run-con. I'm happy to deprecate in favour of cosmicconfig
